<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QWidget" name="widget_9" native="true">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>451</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelreproductor">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
             </property>
             <property name="text">
              <string>Reproduce el audio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelescuchar">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
             </property>
             <property name="text">
              <string>Te escucho</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
             </property>
             <property name="text">
              <string>Escucha tu audio</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>508</width>
            <height>131</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="botonescuchar_3">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>151</width>
             <height>106</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/reproductor.jpg</normaloff>../img/reproductor.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16777215</width>
             <height>98</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="botonplay">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>151</width>
             <height>106</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/reproductor.jpg</normaloff>../img/reproductor.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16777215</width>
             <height>98</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="botonescuchatuaudio">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>10</y>
             <width>151</width>
             <height>106</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/bootnaudiotuyo.png</normaloff>../img/bootnaudiotuyo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16777215</width>
             <height>98</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(206, 206, 206);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="texto1">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btncalificar">
         <property name="text">
          <string>Calificar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
